app/views/Posts/index.html.erb

```diff
@@ -1 +1,36 @@
-<h1>All posts with comments for a given user</h1>
\ No newline at end of file
+<main>
+    <div>
+        <div id="photo">
+        <%= image_tag (@user.photo) %>
+        </div>
+        <div>
+        <h4><%= @user.name %></h4>
+        <p>Number of posts: <%= @user.posts.count %></p>
+        </div>
+    </div>
+    <% if @posts.blank? %>
+    <p>There are not any post currently for this author.</p>
+    <% else %>
+        <ul>
+        <% @posts.each do |post| %>
+        <li class="post">
+            <div class="text">
+            <h4>Post #<%= post.id %></h4>
+                <p><%= post.title %></p>
+                <p>Comments:<%= post.comments_counter %>, Likes: <%= post.likes_counter %></p>
+            </div>
+            <div class="comments">
+                <div class="comments_list">
+                <ul>
+                    <% post.comments.each do |comment| %>
+                    <li><%= @user.name%>: <%= comment.text%></li>
+                    <% end %>
+                </ul>
+                </div>
+            </div>
+        </li>
+        <% end %>
+        </ul>
+    <% end %>
+    <button>Next</button>
+</main>
```

Comment:
Good job working on the project requirements. But is says in the project requirements Note, you should pay more attention on the function as styling is nice to have
The index view page should match the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png)

This is your design:

![Screenshot 2023-06-15 at 9 51 40 AM](https://github.com/ernestmusong/Blog-app/assets/79506105/d32aa112-e904-4d54-838b-d958784cecb0)

Please stick to this design:



![Screenshot 2023-06-15 at 8 32 25 AM](https://github.com/ernestmusong/Blog-app/assets/79506105/6eba96f0-590a-497a-86b5-bd3ae8b3f465)

![Screenshot 2023-06-15 at 8 32 48 AM](https://github.com/ernestmusong/Blog-app/assets/79506105/c42ffc46-445a-41f6-af23-096932112df8)
![Screenshot 2023-06-15 at 8 33 29 AM](https://github.com/ernestmusong/Blog-app/assets/79506105/297f2e88-33ac-4609-b39e-80aee088ea38)

Secondly it says Clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)

Please it would be nice if you implemented this.

Reply:
> Good job working on the project requirements. But is says in the project requirements Note, you should pay more attention on the function as styling is nice to have The index view page should match the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png)
> 
> This is your design:
> 
> ![Screenshot 2023-06-15 at 9 51 40 AM](https://user-images.githubusercontent.com/79506105/246034778-d32aa112-e904-4d54-838b-d958784cecb0.png)
> 
> Please stick to this design:
> 
> ![Screenshot 2023-06-15 at 8 32 25 AM](https://user-images.githubusercontent.com/79506105/246034962-6eba96f0-590a-497a-86b5-bd3ae8b3f465.png)
> 
> ![Screenshot 2023-06-15 at 8 32 48 AM](https://user-images.githubusercontent.com/79506105/246035023-c42ffc46-445a-41f6-af23-096932112df8.png) ![Screenshot 2023-06-15 at 8 33 29 AM](https://user-images.githubusercontent.com/79506105/246035102-297f2e88-33ac-4609-b39e-80aee088ea38.png)
> 
> Secondly it says Clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)
> 
> Please it would be nice if you implemented this.

Thank you @KayLemba for your feedback. I have now implemented the required changes.